tests/phpunit/includes/testcase-trait.php

```diff
@@ -4,17 +4,48 @@
  * A trait to share code between several test case classes.
  */
 trait PLL_UnitTestCase_Trait {
+
+	/**
+	 * @var array|null
+	 */
+	protected $options;
+
+	/**
+	 * @var PLL_Links_Model|null
+	 */
+	protected $links_model;
+
+	/**
+	 * @var PLL_Model|null
+	 */
+	protected $pll_model;
+
+	/**
+	 * @var PLL_Frontend|null
+	 */
+	protected $frontend;
+
+	/**
+	 * @var PLL_Admin|null
+	 */
+	protected $pll_admin;
+
+	/**
+	 * @var PLL_Base|null
+	 */
+	protected $pll_env;
+
 	/**
 	 * A container for Polylang classes instances.
 	 *
-	 * @var object
+	 * @var object|null
```

Comment:
```suggestion
	 * @var PLL_Model|null
```

Reply:
Fixed in [038ed47](https://github.com/polylang/polylang/pull/1143/commits/038ed47eaabc3e3c693b925ce5d1dfc8d29cc26c) with `PLL_Admin_Model`.